js/jquery/jquery.crmProfileSelector.js

```diff
@@ -13,7 +13,7 @@
    * <input type="text" value="{$profileId}" class="crm-profile-selector" />
    * ...
    * cj('.crm-profile-selector').crmProfileSelector({
-   *   groupTypeFilter: "Contact,Individual,Activity;;ActivityType:7",
```

Comment:
Are you sure about this one? It's neither javascript code nor php code. It's a data serialization notation.

Reply:
@colemanw it's also just a code comment - not executable code

Reply:
Agree it's a comment, but according to [this code documentation](https://github.com/civicrm/civicrm-core/blob/5.19/js/model/crm.uf.js#L46) it's actually correct using the double semicolon.

Reply:
@colemanw reverted that change